scripts/main.js

```diff
@@ -0,0 +1,337 @@
+      c1.style.backgroundColor = "aqua";
+      c2.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+    if (boxes[3] === player && boxes[6] === player) {
+      c0.style.backgroundColor = "aqua";
+      c3.style.backgroundColor = "aqua";
+      c6.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations!  ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+    if (boxes[4] === player && boxes[8] === player) {
+      c0.style.backgroundColor = "aqua";
+      c4.style.backgroundColor = "aqua";
+      c8.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+    }
+
+  }
+  //bottom up and across
+  if (boxes[8] === player) {
+    if (boxes[2] === player && boxes[5] === player) {
+      c2.style.backgroundColor = "aqua";
+      c5.style.backgroundColor = "aqua";
+      c8.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+    if (boxes[7] === player && boxes[6] === player) {
+      c6.style.backgroundColor = "aqua";
+      c7.style.backgroundColor = "aqua";
+      c8.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+  }
+  //middle vertical and horizontal
+  if (boxes[4] === player) {
+    if (boxes[3] === player && boxes[5] === player) {
+      c3.style.backgroundColor = "aqua";
+      c4.style.backgroundColor = "aqua";
+      c5.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+    if (boxes[1] === player && boxes[7] === player) {
+      c1.style.backgroundColor = "aqua";
+      c4.style.backgroundColor = "aqua";
+      c7.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+    if (boxes[2] === player && boxes[6] === player) {
+      c2.style.backgroundColor = "aqua";
+      c4.style.backgroundColor = "aqua";
+      c6.style.backgroundColor = "aqua";
+      start.disabled = false;
+      setTimeout(function () {
+        alert(`Congratulations! ${playerName} wins!`);
+        setTime(false)
+        resetGame()
+      }, 100);
+
+    }
+  }
```

Comment:
The first thing that I noticed when looking at all of this repetitive code, would be is there an easier way to do all of this logic? While this is an acceptable way to check for wins, there may be some other shorter / more concise ways to go about this.
Just something to think about if you take on this project again in the future!